libraries/testkit/server.py

```diff
@@ -86,6 +87,13 @@ def delete_bucket(self, name):
         return status
 
     def create_buckets(self, names):
+        # Create a user with username=bucketname
+        if type(names) is list:
```

Comment:
I added a function to delete users and it gets called when bucket gets deleted. We can check if the user exists but not if the password is what we are expecting